src/middleware/onlyAuthenticated.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const jwt = require('jsonwebtoken');
+
+//  error messages
+const sessionNotFoundError = {
+  meesage: 'There is no session; please sign in.',
```

Comment:
You can leave a more human messae to the user, kike you're not signed in please sign in, instead of there's no session which is a bit technical